sqldelight-idea-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -22,14 +22,29 @@
   </project-components>
 
   <extensions defaultExtensionNs="com.intellij">
-    <fileTypeFactory implementation="com.squareup.sqldelight.intellij.lang.SqlDelightFileTypeFactory"/>
```

Comment:
I've removed it as it's deprecated now, and we have to use `<fileType>` for that

Reply:
:+1:

Reply:
good callout. We can remedy that by changing the minimum version for the plugin, it'll prevent old versions of studio/intellij from downloading the plugin

Reply:
hmm, should I squeeze this into the PR? I could also revert that actually, if you don't wanna break old idea clients

Reply:
nah dont worry about it in this pr we can add it before releasing